demo/runtime/src/lib.rs

```diff
@@ -170,6 +171,42 @@ pub type Council = council::Module<Concrete>;
 /// Council voting module for this concrete runtime.
 pub type CouncilVoting = council::voting::Module<Concrete>;
 
+/// Concrete log type.
```

Comment:
could do with more information. consider what it is in relation to other types and why it exists as a type at all.